src/pages/HomePage/Componets/index.jsx

```diff
@@ -0,0 +1,11 @@
+export * from "./Hero/Hero";
+export * from "./AboutUs/AboutUs";
+export * from "./Dashboard/Dashboard";
+export * from "./OurServices/OurServices";
+export * from "./OurCustomers/index";
```

Comment:
Looks good

Now update those components names to be index.jsx, so we can use shorthand `./Hero` (without repetition `./Hero/Hero`)